packages/libs/react-components/src/components/Typography/GradientText.tsx

```diff
@@ -0,0 +1,12 @@
+import { GradientText as StyledText } from './styles';
+
+import React, { FC } from 'react';
+
+export interface IGradientTextProps {
+  as?: 'span';
```

Comment:
do we need this, if there is only one option?

Reply:
no, indeed we can remove that 👍